DXRModules/DeusEx/Classes/DXRLoadouts.uc

```diff
@@ -144,6 +144,15 @@ function CheckConfig()
         item_sets[10].bans = "Lockpick,Multitool";
         item_sets[10].starting_augs = "AugStealth";
 
+        item_sets[11].name = "Explosives Only";
+        item_sets[11].player_message = "Explosives Only!";
+        item_sets[11].bans = "Engine.Weapon,Ammo762mm";
+        item_sets[11].allows = "WeaponGEPGun,WeaponLAW,WeaponLAM,WeaponEMPGrenade,WeaponGasGrenade,WeaponNanoVirusGrenade,WeaponAssaultGun,WeaponRubberBaton";
```

Comment:
When we load a class by string, we assume it's part of the DeusEx package, but the rubber baton is part of our package. For vanilla this doesn't matter, but it does for other mods, so all references inside strings need the package prefix like this with the dot too

```suggestion
        item_sets[11].allows = "WeaponGEPGun,WeaponLAW,WeaponLAM,WeaponEMPGrenade,WeaponGasGrenade,WeaponNanoVirusGrenade,WeaponAssaultGun,#var(package).WeaponRubberBaton";
```